ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -585,6 +584,12 @@ protected Boolean findNextCompactionAndExecute(boolean computeStats) {
     return true;
   }
 
+  @VisibleForTesting
+  public CompactionInfo nextCompactForWorkerAndRuntime() throws TException {
```

Comment:
could we please rename it to 'getNextCompactRequest'

Reply:
It is not really a getter, though. But I can do this minor change.